flow/handler_test.go

```diff
@@ -57,8 +60,29 @@ func TestPostFlow_ShouldAddFlowToRepoForValidRequest(t *testing.T) {
 	assert.Equal(t, expectedFlow, actualFlow)
 }
 
-func TestPostFlow_ShouldReturn400ForInvalidRequest(t *testing.T) {
+func TestPostFlow_ShouldReturn500WhenFlowIsEmpty(t *testing.T) {
+	defer loggertest.Reset()
+	loggertest.Init(loggertest.LogLevelError)
+
+	defer resetFlowRepo()
```

Comment:
I think we can remove this FlowRepo setup. It is not used anywhere, probably a copy & paste.

Reply:
👍